doc/nrf/releases_and_maturity/releases/release-notes-2.6.99-cs.rst

```diff
@@ -0,0 +1,253 @@
+
+The scope of this release is intentionally narrow, concentrating solely on the nRF54L15 device.
+You can use the latest stable release of |NCS| to work with other boards.
+
+All functionality related to nRF54L15 is considered experimental.
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Working with nRF54L Series
+==========================
+
+* Added the :ref:`ug_nrf54l15_gs` page.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+See `Samples`_ for lists of changes for the protocol-related samples.
+
+Thread
+------
+
+* Initial experimental support for nRF54L15 for the Thread CLI and Co-processor samples.
+
+Enhanced ShockBurst (ESB)
+-------------------------
+
+* Added support for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` board.
+
+Applications
+============
+
+This section provides detailed lists of changes by :ref:`application <applications>`.
+
+nRF Desktop
+-----------
+
+* Added:
+
+  * Support for the nRF54L15 PDK with the ``nrf54l15pdk_nrf54l15_cpuapp`` board target.
+    The PDK can act as a sample mouse or keyboard.
+    It supports the Bluetooth LE HID data transport and uses SoftDevice Link Layer with Low Latency Packet Mode (LLPM) enabled.
+    The PDK uses MCUboot bootloader built in the direct-xip mode (``MCUBOOT+XIP``) and supports firmware updates using the :ref:`nrf_desktop_dfu`.
+  * The ``nrfdesktop-wheel-qdec`` DT alias support to :ref:`nrf_desktop_wheel`.
+    You must use the alias to specify the QDEC instance used for scroll wheel, if your board supports multiple QDEC instances (for example ``nrf54l15pdk_nrf54l15_cpuapp``).
+    You do not need to define the alias if your board supports only one QDEC instance, because in that case, the wheel module can rely on the ``qdec`` DT label provided by the board.
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <samples>`.
+
+Bluetooth samples
+-----------------
+
+* :ref:`fast_pair_input_device` sample:
+
+  * Added support for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` board.
+
+* :ref:`peripheral_lbs` sample:
+
+  * Added support for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` board.
+
+* :ref:`bluetooth_central_hids` sample:
+
+  * Added support for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` board.
+
+* :ref:`peripheral_hids_mouse` sample:
+
+  * Added support for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` board.
+
+* :ref:`peripheral_hids_keyboard` sample:
+
+  * Added support for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` board.
+
+* :ref:`central_and_peripheral_hrs` sample:
+
+  * Added support for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` board.
+
+* :ref:`direct_test_mode` sample:
+
+  * Added support for the :ref:`nRF54L15 PDK <ug_nrf54l15_gs>` board.
+
+Bluetooth Mesh samples
+----------------------
+
+* :ref:`bluetooth_mesh_sensor_client` sample:
```

Comment:
Do we want to put it like:


* Added support for the :ref:nRF54L15 PDK <ug_nrf54l15_gs> board in the following samples:

  * Sample1
  * Sample2